doc/hooks/waf.section.md

```diff
@@ -1,3 +1,36 @@
 # wafHook {#wafhook}
 
-Overrides the configure, build, and install phases. This will run the “waf” script used by many projects. If `wafPath` (default `./waf`) doesn’t exist, it will copy the version of waf available in Nixpkgs. `wafFlags` can be used to pass flags to the waf script.
+[Waf](https://waf.io) is a Python-based software building system.
+
+In Nixpkgs, `wafHook` overrides the default configure, build, and install phases.
```

Comment:
Documenting the _lack_ of cross compilation support is missing, a link to https://github.com/NixOS/nixpkgs/issues/248127 would be nice.

Reply:
Curiously, DDGing a bit I found that Samba does explain a bit about cross-compilation:

https://wiki.samba.org/index.php/Waf#cross-compiling

https://wiki.samba.org/index.php/BuildsystemUseAndWhy